<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0033)http://www2.hawaii.edu/~lgarmire/ -->
<html xmlns="http://www.w3.org/1999/xhtml">

<head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <title>Welcome to Lana Garmire's Group</title>

    <link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css" rel="stylesheet">
    <link href="http://fonts.googleapis.com/css?family=Roboto:400,400italic,700,700italic" rel="stylesheet" type="text/css" />
    <style type="text/css">
        body {
            font-size: 16px;
            overflow-y: scroll;
            font-family: Roboto, sans-serif;
        }

        h4 {
            font-size: 1.3em;
        }

        .container {
            margin-top: 20px;
        }

        .row {
            margin-bottom: 20px;
        }

        .card-badge {
            font-size: 5em;
            margin: 20px 30px 30px 30px;
        }

        .panel-body {
            padding: 20px;
        }

        .card-title {
            text-align: center;
        }

        .card-title h4 {
            color: grey;
            margin-bottom: 17px;
        }

        .card-info {
            margin: 10px;
            margin-bottom: 20px;
            font-size: 0.85em;
            color: #999;
        }

        .collapse .in {
            transition: 1s;
        }

        .the-one-toggle:hover .to-be-toggled {
            max-height: 41px;
        }

        .to-be-toggled {
            max-height: 0;
            overflow: hidden;
            transition: max-height 0.25s;
        }

        .footer {
            border-top: 1px solid #ccc;
            padding-top: 3px;
            font-size: 0.6em;
            color: #aaa;
        }

        .footer .info-phrase:hover {
            color: #333;
        }

        .alert a {
            color: inherit;
            font-weight: inherit;
            text-decoration: none;
        }

        .alert a:hover {
            text-decoration: none;
            color: #000;
        }

        #publications h4 {
            margin-top: 30px;
            color: #aaa;
        }

        #members h3 {
            margin-top: 60px;
            margin-bottom: 30px;
            color: #aaa;
        }

        #members .member-photo {
            background-size: auto 100%;
            background-position: center;
            background-repeat: no-repeat;
            background-size: cover;
        }

        #carousel-example-generic {
            height: 600px;
            overflow: hidden;
            margin-bottom: 20px;
        }

        #carousel-example-generic .carousel-inner {
            width: 100%;
            height: 100%;
        }

        #carousel-example-generic .carousel-inner .item {
            background-position: center;
            background-repeat: no-repeat;
            background-size: 100% auto;
            width: 100%;
            height: 100%;
        }

        p.large {
            font-size: 1.1em;
        }
    </style>
    <style type="text/css"></style>
</head>

<body class="twoColLiqLtHdr">
    <div id="container" class="container">
        <nav id="header" class="navbar navbar-default">
            <div class="navbar-header">
                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#main-nav-bar">
            <span class="icon-bar"></span><span class="icon-bar"></span><span class="icon-bar"></span></button>
                <a class="navbar-brand">Lana Garmire Group in Translational Informatics</a></div>
            <div class="collapse navbar-collapse" id="main-nav-bar">
                <ul class="nav navbar-nav navbar-right">
                    <li class="active"><a href="index.html#index" data-toggle="tab">Main</a></li>
                    <li><a href="index.html#photos" data-toggle="tab">Photos</a></li>
                    <li><a href="index.html#members" data-toggle="tab">Members</a></li>
                    <li><a href="index.html#news" data-toggle="tab">News</a></li>
                    <li><a href="index.html#software" data-toggle="tab">Software</a></li>
                    <li><a href="index.html#publications" data-toggle="tab">Publications</a></li>
                </ul>
            </div>
        </nav>
        <div class="tab-content">
            <div class="tab-pane active fade in" id="index">
                <div class="alert alert-dismissable alert-success fade in">
                    <a href="https://www.biorxiv.org/content/early/2018/05/29/166090"><strong>News: 06/30/2018</strong>&nbsp; Dr. Kumardeep Chauhary's first author work on hepatocellular carcinoma (HCC) consesus driver genes' phenotypic impact is accepted by Clinical Cancer Research. Congratulations, Kumar. </a> <br>
                    <a href="https://www.biorxiv.org/content/early/2016/12/11/093021/"><strong>News: 05/16/2018</strong>&nbsp; PhD student Xun Zhu will give oral presentation on our next generation data portal "Granatum X" at SCANGEN meeting of ISMB 2018, Chicago. Good job, Xun! </a><br>
               
                    <a href="https://www.biorxiv.org/content/early/2016/12/11/093021/"><strong>News: 05/16/2018</strong>&nbsp; PhD student Xun Zhu won the fellowship to be selected to attend Cold Spring Harbor Single Cell Analysis Course Training Program, June 30-July 15th. Well done, Xun! </a><br>
                    <a href="https://www.biorxiv.org/content/early/2016/12/11/093021/"><strong>News: 05/15/2018</strong>&nbsp; PhD Alumus Dr. Sijia Huang won the travel fellowship at ICIBM 2018, to present the work in our group "Lilikoi: an R package for personalized pathway-based classification modeling. Congratulations, Sijia! </a><br>
                   
                    <a href="https://www.biorxiv.org/content/early/2016/12/11/093021/"><strong>News: 04/24/2018</strong>&nbsp; PhD student Xun Zhu passed his qualification exam. Congratulations, Xun! </a><br>
               
                    <a href="https://www.biorxiv.org/content/early/2016/12/11/093021/"><strong>News: 03/07/2018</strong>&nbsp; Postdoc Fadhl and Group alumnus Dr. Sijia Huang's co-1st author manuscript: Lilikoi: an R package for personalized pathway-based classification modeling using metabolomics data is accepted by ICIBM 2018. </a><br>
               
                    <a href="https://www.biorxiv.org/content/early/2016/12/11/093021/"><strong>News: 03/07/2018</strong>&nbsp; Group alumnus Dr. Travers Ching's manuscript: Cox-nnet: an artificial neural network Cox regression for prognosis prediction is accepted by PLoS Computational Biology.</a><br>
               
                    <a href="https://labworm.com/blog/top-17-research-tools-of-2017/"><strong>News: 01/04/2018</strong>&nbsp; Our scRNA-Seq platform Granatum has been ranked as No.3 on "TOP 17 research tools of 2017" by LabWorm.</a><br>
                </div>
                <div class="alert alert-dismissable alert-info fade in">
                    <button type="button" class="close" data-dismiss="alert">×</button>
                    <strong>Postdoc and PhD student candidates:</strong> Garmire
                    Group is recruiting competent and motivated postdoc and PhD
                    students who are passionate to cure cancers with the
                    computational biology approach. Previous rigorous training
                    in quantative sciences is required.
                </div>
                <div class="modal fade" id="applyForPosition">
                    <div class="modal-dialog">
                        <div class="modal-content">
                            <div class="modal-header">
                                <button type="button" class="close" data-dismiss="modal"><span
                                                      aria-hidden="true">×</span><span class="sr-only">Close</span></button>
                                <h4 class="modal-title" id="myModalLabel">Application for Garmire's group</h4>
                                <h5 class="modal-title" id="myModalLabel"><span style="font-weight: bold; color: red;">Attention: this is just a demonstration, submission will not be sent!</span></h5>
                            </div>
                            <div class="modal-body">
                                <form class="form-horizontal">
                                    <fieldset>
                                        <div class="form-group"><label for="inputName" class="col-lg-2 control-label">Name</label>

                                            <div class="col-lg-10"><input class="form-control" id="inputName" type="text">
                                            </div>
                                        </div>
                                        <div class="form-group"><label for="inputEmail" class="col-lg-2 control-label">Email</label>

                                            <div class="col-lg-10"><input class="form-control" id="inputEmail" type="text">
                                                <span class="help-block">We will email you when needed.</span>
                                            </div>
                                        </div>
                                        <div class="form-group"><label for="textArea" class="col-lg-2 control-label">Message</label>

                                            <div class="col-lg-10">
                                                <textarea class="form-control" rows="3" id="textArea"></textarea><span class="help-block">Tell
                                                         us your basic
                                                       information, academic
                                                       background and
                                                     interests, as well as your
                                                     future plan.</span>
                                            </div>
                                        </div>
                                        <div class="form-group"><label class="col-lg-2 control-label">Apply for</label>

                                            <div class="col-lg-10">
                                                <div class="radio col-lg-4"><label> <input name="optionsRadios"
                                                                     id="optionsRadios2"
                                                                     value="option2" type="radio">
                              Master </label></div>
                                                <div class="radio col-lg-4"><label> <input name="optionsRadios"
                                                                     id="optionsRadios1"
                                                                     value="option1" checked="checked"
                                                                                     type="radio"> Ph.D </label>
                                                </div>
                                                <div class="radio col-lg-4"><label> <input name="optionsRadios"
                                                                     id="optionsRadios2"
                                                                     value="option2" type="radio">
                              Post Doctor </label></div>
                                            </div>
                                        </div>
                                    </fieldset>
                                </form>
                            </div>
                            <div class="modal-footer">
                                <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
                                <button type="button" class="btn btn-primary" onclick="alert(&#39;Thank you for your application!&#39;)">Submit
                  </button>
                            </div>
                        </div>
                    </div>
                </div>
                <div class="row" style="margin-top: 40px;">
                    <div class="col-md-3">
                        <div id="sidebar1"><img class="panel panel-default" src="http://www.uhcancercenter.org/images/faculty/garmire-lana.jpg" alt="Lana Garmire" height="290"></div>
                    </div>
                    <div class="col-md-4 center-block">
                        <div id="mainContent">
                            <p><strong>Lana Garmire, Ph.D.</strong> (<a href="cv.docx">CV</a>)
                                <a href="https://www.linkedin.com/in/lana-garmire-phd-8829174/"><img src="./social-1_square-linkedin.svg" height="17" /></a>
                                <a href="https://twitter.com/GarmireGroup"><img src="./social-1_square-twitter.svg" height="17" /></a><br>Associate Professor (with tenure)<br> Email: LGarmire at gmail dot com </p>

                            <p><strong>Graduate Faculty</strong><br> Program of Molecular Biosciences and Bioengineering</p>

                     </div>
                    </div>
                    <div class="col-md-5 center-block">
                        <p><strong>Faculty Member</strong><br> Institute of Biogenesis Research</p>
                        <p><strong>Adjunct Faculty</strong><br> Department of Biochemistry, Physiology and Anatomy</p>

                        <p><strong>Adjunct Faculty</strong><br> Department of Obstetrics and Gynecology</p>
                    </div>
                </div>
                <div class="row">
                    <div class="col-md-12">
                        <p class="large">
                            Since September 2012, Dr. Lana Garmire has rapidly risen from a tenure track faculty in translational bioinformatics to a tenured Associate Professor leading a multidisciplinary team of computational and experimental human genomics. Garmire has won numerous competitive federal grant awards of over 7.8 million dollars as the PI, including NIH/NIGMS P20 COBRE (2014-2019), NIH/BD2K K01 award (2014-2019), and two concurrent NIH R01 grant awards from NICHD (2016-2021) and NLM (2016-2020). Her annual direct expenditure is over 1 million per year.
                        </p>
                        <p class="large">
                            Dr. Garmire obtained the MA degree in Statistics (2005) and PhD degree in Comparative Biochemistry (Computational Biology focus, 2007), both from UC-Berkeley. She then did her postdoctoral training (2008-2011) under the joint mentorship of Prof. Shankar Subramaniam in the Bioengineering Dept. and Prof. Christopher Glass in the Department of Cellular and Molecular Medicine, UC-San Diego. In 2012, she won the first bioinformatics NIH SBIR grant for Asuragen Inc (a spin-off of Ambion, the RNA company), and then resumed the tenure-track faculty position since September 2012. Dr. Garmire collaborates with a variety of top researchers nationally and internationally. She has published over 40 papers in high quality journals including Cell and Nature. She has mentored over 30 MD fellows, postdocs, graduate students and undergraduates of various academic backgrounds, in Biology, Mathematics, Phyiscs, (bio)Statistics, Bioengineering, Computer Science and Electrical Engineering. She is an Associate Editor of BMC Bioinformatics and Guest Editor of PLoS Computational Biology.
                        </p>
                    </div>
                </div>
                <div class="row">
                    <div class="col-md-4">
                        <div class="panel panel-default">
                            <div class="panel-body the-one-toggle">
                                <div class="card-title">
                                    <div class="text-center"><span class="glyphicon glyphicon-star card-badge" style="color: rgb(246, 153, 136);"></span></div>
                                    <h4>Research Interests</h4>

                                    <div class="card-info to-be-toggled">The major research interests of Garmire Group
                                    </div>
                                </div>
                                <table class="table table-hover">
                                    <tbody>
                                        <tr>
                                            <td>Translational bioinformatics of cancers and immune-related diseases, such as classifying biomarkers for cancer prognosis and diagnosis </td>
                                        </tr>
                                        <tr>
                                            <td>Use high-throughput methods (next-generation sequencing method etc.) to study non-coding RNAs, such as microRNAs and intergenic non-coding RNAs
                                            </td>
                                        </tr>
                                        <tr>
                                            <td>Integrative omics/clinic data analysis</td>
                                        </tr>
                                        <tr>
                                            <td>single-cell sequencing and bioinformatics</td>
                                        </tr>
                                        <tr>
                                            <td>Develop computational methods to analyze high-throughput data from next-generation sequencing et al. </td>
                                        </tr>
                                        <tr>
                                            <td> Pregnancy adversities and fetal orgin of cancers, public health genomics</td>
                                        </tr>
                                    </tbody>
                                </table>
                            </div>
                        </div>
                    </div>
                    <div class="col-md-4">
                        <div class="panel panel-default">
                            <div class="panel-body the-one-toggle">
                                <div class="card-title">
                                    <div class="text-center"><span class="glyphicon glyphicon-user card-badge" style="color: rgb(174, 213, 129);"></span></div>
                                    <h4>Established Collaborators</h4>

                                    <div class="card-info to-be-toggled">Major established collaborators
                                    </div>
                                </div>
                                <table class="table table-hover">
                                    <tbody>
                                        <tr>
                                            <td><strong>Yale University:</strong><br> Dr. Sherman Weissman (Sterling Professor and a member of National Academy of Sciences)<br> Dr. Rong Fan (Associate Professor of Biomedical Engineering)<br> Dr. Richard Flavell (Sterling Professor, a member of National Academy of Sciences, and Chair of Immunology Department)
                                            </td>
                                        </tr>
                                        <tr>
                                            <td><strong>University of Pennsyvinia:</strong><br> Dr. Jason Moore (Founding Director of Institute for Biomedical Informatics and an AAAS member)
                                            </td>
                                        </tr>
                                        <tr>
                                            <td><strong>University of Hawaii School of
                            Medicine:</strong><br> Dr. William Boisvert<br> Drs. Dena Towner & Men-Jean Lee (OBGYNs)
                                                <br> Dr. Steven Ward (Director of Institute of Biogenesis Research)<br> Dr. Monika Ward (Institute of Biogenesis Research)<br> Dr. Ben Fogelgren (Department of Biochemistry, Physiology and Anatomy) </td>
                                        </tr>
                                        <tr>
                                            <td><strong>University of Hawaii Cancer Center:
                          </strong><br>Dr. Wei Jia (Associate Director of Shared Resources)<br> Dr. Herbert Yu (Director of Epidemiology Program)<br> Dr. Charles Rosser (Director of Clinical and Translational Research Program) </td>
                                        </tr>
                                    </tbody>
                                </table>
                            </div>
                        </div>
                    </div>
                    <div class="col-md-4">
                        <div class="panel panel-default">
                            <div class="panel-body the-one-toggle">
                                <div class="card-title">
                                    <div class="text-center"><span class="glyphicon glyphicon-globe card-badge" style="color: rgb(145, 167, 255);"></span></div>
                                    <h4>Establishing Collaborators</h4>

                                    <div class="card-info to-be-toggled">Other establishing collaborators (with active projects and/or papers to be submitted together)
                                    </div>
                                </div>
                                <table class="table table-hover">
                                    <tbody>
                                        <tr>
                                            <td><strong>Yale University:</strong><br> Dr. Shangqin Guo (Assistant Professor of Yale Stem Cell Center)
                                            </td>
                                        </tr>
                                        <tr>
                                            <td><strong>University of Hawaii:</strong><br>Drs. Linda Chang and Thomas Ernest at Neuroscience and MRI Program.
                                                <br> Dr. Jason Leigh, Department of Information and Computer Science.</td>
                                        </tr>
                                    </tbody>
                                </table>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            <div class="tab-pane fade" id="photos">
                <h3>Photos</h3>

                <div id="carousel-example-generic" class="carousel slide" data-ride="carousel" data-interval="false">
                    <!-- Indicators -->
                    <ol class="carousel-indicators">
                        <li data-target="#carousel-example-generic" data-slide-to="0" class="active"></li>
                        <li data-target="#carousel-example-generic" data-slide-to="1"></li>
                        <li data-target="#carousel-example-generic" data-slide-to="2"></li>
                        <li data-target="#carousel-example-generic" data-slide-to="3"></li>
                        <li data-target="#carousel-example-generic" data-slide-to="4"></li>
                    </ol>
                    <!-- Wrapper for slides -->
                    <div class="carousel-inner">
                        <div class="item" style="background-image: url(member_photos/travers_defense.jpg);">
                            <div class="carousel-caption">
                                Dr. Garmire giving an emotional speech on Travers's dissertation defense
                            </div>
                        </div>
                        <div class="item" style="background-image: url(member_photos/dylan_posters.jpg);">
                            <div class="carousel-caption">
                                Our undergraduate intern Dylan is presenting his poster
                            </div>
                        </div>
                        <div class="item" style="background-image: url(member_photos/potluck_20161118_2.jpeg);">
                            <div class="carousel-caption">
                                Group members having a cheerful potlock welcoming new postdocs
                            </div>
                        </div>
                        <div class="item" style="background-image: url(member_photos/conference_room.jpg);">
                            <div class="carousel-caption"> Happy moment was captured by the camera on the conference room when group members finished a heated discussion
                            </div>
                        </div>
                        <div class="item active" style="background-image: url(member_photos/COBRE_gang.jpg);">
                            <div class="carousel-caption"> Together we are a unit: the COBRE P20 PI and Project Leaders in IDeA meeting, 2015
                            </div>
                        </div>
                    </div>
                    <!-- Controls -->
                    <a class="left carousel-control" href="index.html#carousel-example-generic" role="button" data-slide="prev"> <span class="glyphicon glyphicon-chevron-left"></span> </a>
                    <a class="right carousel-control" href="index.html#carousel-example-generic" role="button" data-slide="next"> <span class="glyphicon glyphicon-chevron-right"></span> </a>
                </div>
            </div>
            <div class="tab-pane fade" id="members">
                <h3>Study manager</h3>
                <div class="row">
                    <div class="col-md-3">
                        <div class="well member-photo" style="height: 300px; background-image: url(member_photos/cameron_lassiter.png);"></div>
                        <h4>Cameron Lassiter</h4>

                        <p>Cameron obtained his BS degree in Physiology and Neurobiology from the University of Connecticut, and an MS from John Hopkins University and the University of Maryland, Baltimore in Bioinformatics and Biomedical Sciences, respectively. He has previously worked in both industry and academia on a mixture of pre-clinical and clinical projects; including chemotherapy-induced peripheral neuropathy, lupus, cerebral ataxia, and hepatitis C; involving omics data sets and wet-lab validation. He is interested in bridging the gap between computational and experimental knowledge.</p>

                        <p> Email: CLassiter@cc.hawaii.edu </p>
                    </div>
                </div>

                <h3>Postdoc</h3>
                <div class="row">
                    <div class="col-md-3">
                        <div class="well member-photo" style="height: 300px; background-image: url(member_photos/olivier_poirion.jpg);"></div>
                        <h4>Olivier Poirion</h4>

                        <p>Olivier obtained an Engineering degree in Bioprocess Sciences with minor in Bioinformatics from Ecole Nationale Superieure d'Agronomie et des Industries Alimentaires (ENSAIA), France and a Ph.D. in Evolutionary Genomics, delivered by the Ecole Centrale de Lyon (ECL), France. Olivier is interested in machine learning and datamining analyses, applied to genomics.</p>

                        <p> Email: opoirion@hawaii.edu </p>
                    </div>
                    <div class="col-md-3">
                        <div class="well member-photo" style="height: 300px; background-image: url(member_photos/fadhl_alakwaa.jpg);"></div>
                        <h4>Fadhl Alakwaa</h4>

                        <p>Fadhl Alakwaa obtained his Master’s degree in Biomedical engineering at Cairo University in Giza, Egypt. He then completed his PhD in the same department, working on extracting biological knowledge from biological databases. His research was mainly on modeling human gene-gene interaction using Bayesian networks and biclustering. His research interests include but are not limited to Genomic Cancer Biology, Bioinformatics, Gene Regulatory Network, Biological Networks, Molecular Biology, and Biotechnology.
                        </p>

                        <p> Email: falakwaa@cc.hawaii.edu </p>
                    </div>
                    <div class="col-md-3">
                        <div class="well member-photo" style="height: 300px; background-image: url(member_photos/paula_benny.jpg);"></div>
                        <h4>Paula Benny</h4>

                        <p>Paula obtained her B.Sc (Hons) in Life Sciences from the National University of Singapore. She also has a M.Sc in Biomedical Sciences and a PhD in Biochemistry. Her research interests include genetics, pathophysiology and cancer.</p>

                        <p> Email: pbenny@cc.hawaii.edu </p>
                    </div>
                    <div class="col-md-3">
                        <div class="well member-photo" style="height: 300px; background-image: url(member_photos/ryan_schuleter.jpg);"></div>
                        <h4>Ryan Schlueter</h4>

                        <p>Ryan Schlueter obtained his BS in biological sciences from the College of William and Mary and his medical degree (D.O.) from the Virginia College of Osteopathic Medicine in Blacksburg, VA. He then completed residency training in Obstetrics and Gynecology at the University of Buffalo, SUNY. Currently he is a fellow at the University of Hawaii in Maternal-Fetal Medicine. Research interests include maternal obesity and preeclampsia, stillbirth, and modifiable risk factors for maternal disease.</p>

                        <p> Email: rjschlue@hawaii.edu </p>
                    </div>
                </div>

                <h3>Graduate Student</h3>
                <div class="row">
                    <div class="col-md-3">
                        <div class="well member-photo" style="height: 300px; background-image: url(member_photos/xun_zhu.jpg);"></div>
                        <h4>Xun Zhu</h4>

                        <p>Xun obtained his BS degree in Applied Mathematics from Tianjing Polytechnic University, China, and MS degree in Applied Mathematics from University of Southern California. He is currently working on single-cell RNA-seq analysis. </p>

                        <p> Email: xunzhu@cc.hawaii.edu </p>
                    </div>
                    <div class="col-md-3">
                        <div class="well member-photo" style="height: 300px; background-image: url(member_photos/cedric_arisdakessian.jpg);"></div>
                        <h4>Cédric Arisdakessian</h4>

                        <p>Cedric received his Engineering degree from the Ecole Centrale de Lyon (ECL) and specialized in Computer Sciences. In addition, he obtained his MS degree in Biomathematics, Biostatistics, Bioinformatics and Public Health from Lyon I university. His research interests include machine learning approaches for biomarker discovery and predictive analyses using multiomics data.
                        </p>

                        <p> Email: carisdak@hawaii.edu </p>
                    </div>
                </div>

                <h3>Software Engineer</h3>

                <div class="row">
                    <div class="col-md-3">
                        <div class="well member-photo" style="height: 300px; background-image: url(member_photos/thomas_wolfgruber.jpg);"></div>
                        <h4>Thomas Wolfgruber</h4>

                        <p>Thomas received his BS degree in Computer Engineering from Santa Clara University and PhD in Molecular Biosciences and Bioengineering at the University of Hawaii at Manoa. He has researched the genomics and epigenomics of centromeres, and is interested in making software for research.</p>

                        <p> Email: twolfgruber@cc.hawaii.edu </p>
                    </div>

                    <div class="col-md-3">
                        <div class="well member-photo" style="height: 300px; background-image: url(member_photos/breck_yunits.png);"></div>
                        <h4>Breck Yunits</h4>

                        <p>Breck obtained his BS degree in Economics from Duke University. He previously worked at Microsoft on cloud computing and did research on programming languages and data visualization. He is interested in machine learning applied to genomics and biomarkers.</p>

                        <p>byunits@cc.hawaii.edu</p>
                    </div>
                </div>

                <h3>Industrial Consultant</h3>

                <div class="row">
                    <div class="col-md-3">
                        <div class="well member-photo" style="height: 300px; background-image: url(member_photos/mike_kubal.jpg);"></div>
                        <h4>Mike Kubal</h4>

                        <p>Michael earned his MS in Computer Science from the
                        University of Chicago. He enjoys working at the
                        interface of biology and software with commercial and
                        academic teams on a wide variety of bioinformatics
                        projects. His passion is finding creative solutions that
                        leverage machine learning and immersive
                        technologies.</p>

                        <p> Email: mikekubal@yahoo.com </p>
                    </div>
                </div>



                <h3>Alumni</h3>
                <div class="row">
                    <div class="col-md-3">
                        <h4>Sijia Huang</h4>

                        <p>Sijia Huang, PhD (Aug. 2013 - Dec. 2017) is current a postdoc researcher in the group of Prof. Jason Moore, Director of Biomedical Informatics Insitute at U Penn. She obtained her BS degree in Financial Statistics in HuaZhong University of Science and Technology in China, and MS degree in biostatistics from University of Florida. Her PhD thesis was in biomarker classification and genomic/clinic data integration in the context of breast cancers.</p>

                        <p> Email: shuang@cc.hawaii.edu </p>
                    </div>
                    <div class="col-md-3">
                        <h4>Kumardeep Chaudhary</h4>

                        <p>Kumardeep, Postdoc (April 2016- Dec 2017). Kumar is current a senior postdoc in School of Medicine, Mount Sainai. He was a postdoc in Garmire group and relocated for family reason, he did his BSc (Gold Medal) in Zoology (Hons.) from Panjab University, Chandigarh (India) followed by MSc in Systems Biology and Bioinformatics from the same University. He earned his Doctorate from Bioinformatics Centre at Institute of Microbial Technology, Chandigarh (IMTECH-JNU PhD program). His broad interests include: Personalized Medicine, Next Generation Sequencing, Biomaker identification, survival analysis, Bioinformatics, Computational Biology, GWAS and Medical Health. His ultimate goal is to improvise upon translational health for the benefit of mankind using high-throughput data.</p>

                        <p> Email: kchaudhary@cc.hawaii.edu </p>
                    </div>
                    <div class="col-md-3">
                        <h4>Travers Ching</h4>

                        <p>Travers, PhD (2013-2017) is currently a computational biologist in Adaptive Biotechnologies Inc, Seattle. Travers graduated with PhD in May 2017 from Garmire group. He obtained his BS degree in Applied Physics with minor in Biomedical Engineering from Cornell University, and MS degree in microbiology from University of Hawaii at Manoa. He has broad interest in methylation data analysis, RNA-Seq sample size estimation, and integration of RNA-Seq, methylation data in breast cancer and lung cancers.
                        </p>

                        <p> Email: tching@cc.hawaii.edu </p>
                    </div>
                    <div class="col-md-3">
                        <h4>Michael Ortega</h4>

                        <p>Michael obtained his PhD in Developmental and Reproductive Biology with the Institute for Biogenesis Research at the University of Hawaii while working on replication and DNA damage in gametes and preimplantation embryos. His current research interests include understanding human health and disease by utilizing high-throughput sequencing and multi-omics approaches to investigate critical issues in RNA and cancer biology. He is concentrated in exploring these issues by developing methods for single-cell research and data analysis.</p>

                        <p> Email: maortega@hawaii.edu </p>
                    </div>
                </div>
                <div class="row">
                    <div class="col-md-3">
                        <h4>Joshua Chen</h4>

                        <p>Joshua obtained his BS degree in Biomedical Engineering from the University of Miami and is currently working on his MS in Electrical Engineering at the University of Hawaii at Manoa. He plans on attending medical school afterwards. He is currently working on a miRNA project.</p>

                        <p> Email: jschen2@hawaii.edu </p>
                    </div>
                    <div class="col-md-3">
                        <h4>Austin Tasato</h4>

                        <p>Austin Tasato majors in Electrical Engineering at University of Hawaii at Manoa. He worked with Xun Zhu on developing Granatum, an scRNA-seq analysis online platform.</p>

                        <p> Email: atasato@hawaii.edu </p>
                    </div>
                    <div class="col-md-3">
                        <h4>Runmin Wei</h4>

                        <p>Runmin obtained his MS degree in Pharmaceutical Science from Shanghai Jiaotong University, China.</p>

                        <p> Email: rwei@cc.hawaii.edu </p>
                    </div>
                    <div class="col-md-3">
                        <h4>Jonathan Uejbe</h4>

                        <p>Jonathan Uejbe is an undergraduate student pursuing his Bachelor's of Science in Electrical Engineering. His interests include new computational methods and algorithms in data analysis. He currently is working on new computational techniques to analyze GWAS data.</p>

                        <p> Email: uejbe@cc.hawaii.edu </p>
                    </div>
                </div>
                <div class="row">
                    <div class="col-md-3">
                        <h4>Nicole Chong</h4>

                        <p>Nicole Chong is a 3rd year undergraduate student with 4.0 GPA majoring in Biology (with Mathematical Biology Certificate) at University of Hawaii at Manoa. She is working on breast cancer biomarker classification project in collaboration with Sijia Huang, using an integrative metabolomics and transcriptomics approach.</p>

                        <p> Email: nchong6@hawaii.edu </p>
                    </div>
                    <div class="col-md-3">
                        <h4>Jeffery Li</h4>

                        <p>Jeffery Li is a sophomore with 4.0 GPA majoring in Biomedical Engineering at John Hopkins University. A Punahou graduate and local Hawaiian, he returns to UH for summer session, and is working on prediction of global gene expression using gene methylation patterns.</p>
                    </div>
                    <div class="col-md-3">
                        <h4>James Ha</h4>

                        <p>James Ha is a sophomore with 4.0 GPA majoring in Biology at Caltech. A Punahou graduate and local Hawaiian, he returns to UH for summer session, and is working on global methylation changes of cord blood in pre-eclampsia.</p>
                    </div>
                    <div class="col-md-3">
                        <h4>Mark Menor</h4>

                        <p>Information and Computer Science Department, University of Hawaii at Manoa.</p>
                    </div>
                </div>
                <div class="row">
                    <div class="col-md-3">
                        <h4>Cameron Yee</h4>

                        <p>Undergraduate intern, Neuroscience Major, University of Washington.
                        </p>
                    </div>
                    <div class="col-md-3">
                        <h4>Liangqun Lu</h4>

                        <p>Liangqun obtained her BS degree in Biological Sciences and MS degree in Bioinformatics, both from China Agricultural University.</p>

                        <p> Email: llu@cc.hawaii.edu </p>
                    </div>
                </div>
            </div>
            <div class="tab-pane fade" id="news">
                <div id="mainContent">
                    <h2> News </h2>
                    <ul>
                    <a href="about:blank"><strong>News: 12/14/2017</strong>&nbsp; Olivier and Kumar's paper "Deep Learning data integration for better risk stratification models of bladder cancer" is accepted as an oral presentation by AMIA joint summit meeting 2018. </a><br>
                    <a href="about:blank"><strong>News: 12/13/2017 </strong> &nbsp; Welcome Electricial Engineering graduate student Shane Spencer join our group! </a><br>
                    <a href="about:blank"><strong>News: 12/06/2017</strong>&nbsp; Our review "Using Single-Cell Multiple Omics Approaches to Resolve Tumor Heterogeneity" is accepted by Clinical and Translational Medicine (publisher: Springer). </a><br>
                    <a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-017-0492-3"><strong>News: 12/05/2017</strong>&nbsp; Xun's paper "Granatum: a graphical single-cell RNA-Seq analysis pipeline for genomics scientists" is published today on Genome Medicine. </a><br>
                    <a href="about:blank"><strong>News: 11/01/2017</strong>&nbsp; Fadhl's manuscript "Deep learning accurately predicts estrogen receptor status in breast cancer metabolomics data" is accepted by Journal of Proteome Research. </a><br>
                    <a href="about:blank"><strong>News: 10/30/2017</strong>&nbsp; Xun's manuscript "Granatum: a graphical single-cell RNA-Seq analysis pipeline for genomics scientists" is accepted by Genome Medicine. </a><br>
                    <a href="about:blank"><strong>News: 10/30/2017</strong>&nbsp; PhD student Sijia Huang successfully defended her PhD thesis! </a><br>
                    <a href="about:blank"><strong>News: 09/15/2017</strong>&nbsp; Kumar and Olivier's co-1st author manuscript "Deep Learning based multi-omics integration robustly predicts survivals in liver cancer" is accepted by AACR's Clinical Cancer Research. </a><br>
                    <a href="about:blank"><strong>News: 09/15/2017</strong>&nbsp; Dr. Travers Ching's paper "Pan-cancer analysis of expressed single nucleotide variants in long inter-genic non-coding RNA" is accepted by PSB 2018. </a><br>
                    <a href="about:blank"><strong>News: 08/06/2017</strong>&nbsp; Postdoc Fadhl's paper "Deep learning predicts estrogen receptor status in breast cancer metabolomics data" is accepted by TBC 2017. </a><br>
                    <a href="about:blank"><strong>News: </strong> &nbsp; Welcome new PhD students Cedric Arisdakessian join our group! </a><br>
                    <a href="about:blank"><strong>News: 07/03/2017</strong>&nbsp; Dr. Garmire, Xun Zhu and the team's work on single cell bioinformatics have been profiled in a Nature technology review article. </a><br>
                    <a href="about:blank"><strong>News: 07/03/2017</strong>&nbsp; Dr. Garmire was awarded a faculty travel award at 12th International Conference on Pathways, Networks and Systems Medicine. </a><br>
                    <a href="about:blank"><strong>News: 06/02/2017</strong>&nbsp; Dr. Garmire has received tenure and been promoted to Associate Professor. </a><br>
                    <a href="about:blank"><strong>News: 06/02/2017</strong>&nbsp; Our review paper: More is better: recent progress in multi-omics data integration is published by Frontiers in Genetics. </a><br>
                    <a href="about:blank"><strong>News: 05/22/2017</strong>&nbsp; Dr. Garmire will be presenting at 12th International Conference on Pathways, Networks and Systems Medicine, Heraklion, Crete, Greece. </a><br>
                    <a href="about:blank"><strong>News: 05/22/2017</strong>&nbsp; Dr. Garmire will be attending GCAT study section panel review in June. </a><br>
                    <a href="about:blank"><strong>News: 05/15/2017</strong>&nbsp; Our collaborative work between Dr. Travers Ching, Dr. Garmire and Dr. Norm Lee's group at George Washington University: "Aberrant Alternative Splicing in African American Prostate Cancer: novel driver of tumor aggressiveness and drug resistance", is accepted by Nature Communications!</a><br>
                    <a href="about:blank"><strong>News: 05/02/2017</strong>&nbsp; The first PhD student in our group, Travers Ching, has successfully defended his PhD thesis! Congratulations Dr. Ching! .</a><br>
                    <a href="about:blank"><strong>News: 04/14/2017</strong>&nbsp; Our collaborative paper with Dr. Peiwen Fei's group at UH Cancer Center is accepted by Oncogene.</a><br>
                    <a href="about:blank"><strong>News: 03/20/2017</strong>&nbsp; Dr. Garmire is selected to receive 1 of the 3 seed grants from UHCC,
              to develop linc biomarkers in liver cancer.</a><br>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/28111977"><strong>News:
                03/09/2017</strong>&nbsp; The parasite award committee's Letter
              to Editor, "Celebrate parasites", is accepted by Nature
               Genetics.</a><br>
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/28111977"><strong>News:
                01/26/2017</strong>&nbsp;The paper with our
              collaborators at Shanghai Jiaotong University, "Genome-wide analysis of DNA methylation and their
              associations with long noncoding RNA/mRNA expression in
              non-small-cell lung cancer", is accepted by Epigenomics.</a><br>

                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/28003475"><strong>News:
                01/01/2017</strong>&nbsp;The paper with our
              collaborator Dr. Sherman Weissman at Yale,
              "Single cell transcriptomics reveals unanticipated features of
              early hematopoietic precursors", is published on Nucleic Acids
              Research.</a><br>

                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/27897014"><strong>News:
                  12/08/2016</strong>&nbsp;Dr. Garmire's paper "The training of
                next generation data scientists in biomedicine" is accepted by
                Pacific Symposium on Biocomputing.</a><br>

                        <a href="https://peerj.com/preprints/1839/"><strong>News:
                  12/08/2016</strong>&nbsp;Xun's paper "Detecting heterogeneity
                in single-cell RNA-Seq data by non-negative matrix factorization"
                is accepted by PeerJ.</a><br>

                        <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4065187/"><strong>News:
                  10/27/2016</strong>&nbsp;Travers's co-first author paper "Integrated
                Analysis of DNA Methylation, LncRNA and Gene Expression in
                Non-Small Cell Lung Cancer" is accepted by Scientific
              Reports.</a><br>

                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/27981970"><strong>News:
                  10/26/2016</strong>&nbsp;Liangqun's paper "Time Series
                miRNA-mRNA integrated analysis reveals critical miRNAs and
                targets in macrophage polarization" is accepted by Scientific
              Reports.</a><br>

                        <a href="http://www.uhcancercenter.org/"><strong>News:
                  10/21/2016</strong>&nbsp;Drs. Paula Benny, Thomas Wolfgruber
                and Fadhl Al-Akwaa will be joining the group, welcome!</a><br>

                        <a href="http://www.uhcancercenter.org/"><strong>News:
                  08/19/2016</strong>&nbsp;Dr. Garmire has been awarded a second
                competitive R01 grant from National Institute of Child Health and
                Human Development to discover mulit-omics biomarkers for
              preeclampsia and breast cancer risks.</a><br>

                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/27708664"><strong>News:
                  08/19/2016</strong>&nbsp;Our review paper "Single-cell
                Transcriptomics Bioinformatics and Computational Challenges" is
                accepted by Frontiers in Genetics.</a><br>

                        <a href="http://www.uhcancercenter.org/"><strong>News:
                  08/13/2016</strong>&nbsp;Dr. Garmire is invited by NIH to
                review in the BDMA study section.</a><br>

                        <a href="http://www.uhcancercenter.org/"><strong>News:
                  07/26/2016</strong>&nbsp;Dr. Garmire has been awarded a
                competitive R01 grant from National Library of Medicine to
                develop integrative bioinformatics platform for single-cell
              genomics analysis.</a><br>

                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/27383773"><strong>News:
                  06/25/2016</strong>&nbsp;PhD student Sijia Huang's 1st-author
                paper "A nomogram derived by combination of demographic and
                biomarker data improves the non-invasive evaluation of patients
              at risk for bladder cancer" is accepted by Cancer Epidemiology,
              Biomarkers and Prevention.</a><br>

                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/27409342"><strong>News:
                  05/03/2016</strong>&nbsp;Runmin Wei's paper “Meta-dimensional
                data integration identifies critical pathways for susceptibility,
                tumorigenesis and progression of endometrial cancer" is accepted
              by Oncotarget.</a><br>

                        <a href="http://www.uhcancercenter.org/"><strong>News:
                  04/12/2016</strong>&nbsp;Dr. Garmire is invited by NIH as a
                panel reviewer.</a><br>

                        <a href="http://www.uhcancercenter.org/"><strong>News:
                  04/12/2016</strong>&nbsp;Dr. Garmire is invited to serve on the
                committee for PSB "Parasite Award" for promoting rigorous
                secondary data analysis.</a><br>

                        <a href="http://www.uhcancercenter.org/"><strong>News:
                  12/06/2015</strong>&nbsp;Two new postdocs, Dr. Kumardeep
                Chaudhary and Dr. Oliver Poirion will be joining our group.
                Welcome, Kumardeep and Oliver! </a><br>

                        <a href="http://www.uhcancercenter.org/"><strong>News:
                  12/06/2015</strong>&nbsp;Our review paper on lincRNAs is
                accepted by BioData Mining </a><br>

                        <li> 03/19/2015, our collaborative paper with Prof. Richard Flavell's group from Yale is accepted by Nature! </li>

                        <li> 12/2014, Dr. Lana Garmire obtained wet-lab space at UHCC </li>

                        <li> 11/13/2014, Dr. Lana Garmire won the "best paper of the month" at UHCC for the paper" </li>

                        <li> 09/07/2014, Dr. Lana Garmire received $933,000 NIH K01 mentored scientist career award to conduct research in Big Data Science. </li>

                        <li> 07/01/2014, $10.8 million COBRE P20 reproductive research grant awarded to UH medical school. Dr. Lana Garmire is one of the four project leaders.</li>

                        <li> July 1st, 2014, Dr. Garmire receives the NIH cardiovascular COBRE pilot funding, in collaboration with Dr. Will Boisvert Lab at UH JABSOM </li>

                        <li><a href="http://www.uhcancercenter.org/index.php/about-us/news-releases/306">
                  May 16th, 2014, Travers Ching receives "the Best Poster
              Award" in the 2014 Weinman Symposium, University of Hawaii Cancer
              Center</a></li>

                        <li> Aug, 2013. Traver Ching joins Garmire group for PhD study, welcome! </li>

                        <li> Aug. 2013. Sijia Huang joins Garmire group for PhD study, welcome! </li>

                        <li> Oct 2nd, 2013. Dr. Garmire gave a talk on RNA-Seq power analysis and sample size estimation at Cold Spring Harbor Meeting, Asia </li>

                        <li> Dec. 15th, Dr. Garmire receives the Collaboration Enhancement Award from RTRN Small Grants Program, to enhance the collaboration with Dr. Sherman Weissman's Lab at Yale University </li>
                    </ul>
                </div>
            </div>
            <div class="tab-pane fade" id="publications">
                <div id="mainContent">
                    <h2>Publications</h2>
                    <h4><strong>2018</strong></h4>
                     <div class="publication-item">
                         <p>53. <strong>Arisdakessian C, Poirion O, Yunits B, Zhu X, Garmire LX </strong>. DeepImpute: an accurate, fast and scalable deep neural network method to impute single-cell RNA-Seq data. submitted.   

                       (<a href="https://www.biorxiv.org/content/early/2018/06/22/353607">preprint</a>)

                        </p>   

                        <div data-doi="10.1101/353607" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>

                    </div>

                    <div class="publication-item">
                        <p>52. Stein-O' Brien, Arora R, Culhane AC, Favorov A, <strong>Garmire LX </strong>, Greene C, Goff LA, Li Y, Ngom A, Yanxun Xu Y, Fertig EJ. Entering the matrix: factorization uncovers knowledge from omics. in revision.   
                        (<a href="https://www.biorxiv.org/content/early/04/02/196915"> preprint</a>)
                        </p>   
                        <div data-doi="10.1101/196915" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>
                    <div class="publication-item">
                        
                        <p>51. <strong>Alakwaa F, Huang S, Yunits B and Garmire LX </strong>. Lilikoi: an R package for personalized pathway-based classification modeling using metabolomics data, in revision.  
                       <a href="https://www.biorxiv.org/content/early/2018/03/16/283408"> (preprint</a>)
                        </p>   
                        <div data-doi="10.1101/283408" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>
                    
                     <div class="publication-item">
                         
                        <p>50. <strong>Schlueter, RJ, Al-Akwaa FM,</strong> Benny PA, Gurary A, Xie G, Jia W, <strong>Chun X, Chern I, Garmire L</strong>, Metabolomics profile of cord blood is associated with maternal pre-pregnant obesity in a prospective multi-ethnic cohort displaying health disparities, under review. (
                            <a href="http://biorxiv.org/content/early/2018/02/13/264374.1"> (preprint</a>)
                        </p>
                        <div data-doi="10.1101/264374" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>
                     <div class="publication-item">
                        <p>49. <strong>Ching T, Zhu X, Garmire LX</strong>. Cox-nnet: an artificial neural network Cox regression for prognosis prediction, accepted, PLoS Comp Biol. (
                            <a href="http:http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006076" (link </a>)
                        </p>
                        <div data-doi="10.1371/journal.pcbi.1006076" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>
                    <div class="publication-item">
                    <p>48. <strong>Chaudhary K, Lu L, Poirion O, Garmire LX </strong>, Multi-modal meta-analysis of 1494 hepatocellular carcinoma samples reveals vast impacts of consensus driver genes on phenotypes. in revision. (
                            <a href="http://www.biorxiv.org/content/early/2017/12/30/166090">preprint</a>)
                        </p>
                        <div data-doi="10.1101/166090" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>
                    <p>47. Yamasato K, <strong>Ching T</strong>, <strong>Garmire
              LX</strong>, Berry M, Towner D, Evaluation of a maternal cardiovascular gene array in early on-set preeclampia in a dominantly Asian cohort, submitted.
                    </p>

                    <div class="publication-item">
                        <p>46. <strong>Poirion O</strong>, <strong>Zhu X</strong>,
                            <strong>Ching T</strong>, <strong>Garmire L</strong> Using Single Nucleotide Variations of Single-Cell RNA-Seq to identify tumor Subpopulation and Genotype-phenotype Links. in revision. (
                            <a href="http://biorxiv.org/content/early/2017/03/01/095810">preprint</a>)
                        </p>
                        <div data-doi="10.1101/095810" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>
                    <h4><strong>2017</strong></h4>

                    <div class="publication-item">
                        <p>45. <strong> Poirion O, Chaudhary K, Garmire LX </strong>, Deep Learning data integration for better risk stratification models of bladder cancer. Accepted. AMIA joint summit meeting, 2018 </p>
                    </div>

                    <div class="publication-item">
                        <p>44. <strong> Ortega M, Poirion O, Zhu X, Huang SJ </strong>, Sebra R, <strong> Garmire LX </strong>, Using Single-Cell Multiple Omics Approaches to Resolve Tumor Heterogeneity. Clinical and Translational Medicine (Springer). 2017. 6(1):46 </p>
                       (<a href="https://www.ncbi.nlm.nih.gov/pubmed/29285690">link</a>)
                    </div>

                    <div class="publication-item">
                        <p>43. <strong>Ching T, Garmire LX </strong>, Pan-cancer analysis of expressed single nucleotide variants in long inter-genic non-coding RNA. Pac Symp Biocomput. 2018;23:512-523. </p>
                        (<a href="https://www.ncbi.nlm.nih.gov/pubmed/29218910">link </a>)
                        <div data-doi="10.1186/s40169-017-0177-y" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>42. <strong>Al-Akwaa F, Chaudhary K, Garmire LX </strong>, Deep learning predicts estrogen receptor status in breast cancer metabolomics data. Journal of Proteome Research.
                        (<a href="http://pubs.acs.org/doi/abs/10.1021/acs.jproteome.7b00595">link</a>)</p>
                        <div data-doi="10.1101/214254" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>41. <strong>Huang SJ</strong>, <strong>Chaudhary K</strong>,
                            <strong>Garmire LX</strong>, More is better: recent progress in multi-omics data integration, accepted, Frontiers in Genetics. </p>
                        (<a href="https://www.ncbi.nlm.nih.gov/pubmed/28670325">link</a>)
                        </p>
                        <div data-pmid="28670325" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>40. <strong>Zhu X</strong>, <strong>Wolfgruber T</strong>,
                            <strong>Tasato A, Arisdakessian C</strong>, Garmie D, <strong>Garmire LX</strong>, Granatum: A graphical single cell RNA-Seq analysis pipeline for genomics scientists, Genome Medicine, 9(1):108. (
                            <a href="https://www.ncbi.nlm.nih.gov/pubmed/29202807">link</a>)
                        </p>
                        <div data-doi="10.1101/110759" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>39. <strong>Chaudhary K</strong>, <strong>Poirion O</strong>,
                            <strong>Lu L</strong>, <strong>Garmire LX</strong>, Deep Learning based multi-omics integration robustly predicts survivals in liver cancer. Clinical Cancer Research (
                            <a href="https://www.ncbi.nlm.nih.gov/pubmed/28982688">link</a>)
                        </p>
                        <div data-doi="10.1101/114892" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>38. Wang BD, Ceniccola K, Hwang S, Andrawis R, Horvath A, Freeman JA, Knapp S, <strong>Ching T</strong>, <strong>Garmire
                LX</strong>, Pate lV, Garcia-Blanco MA, Patierno SR, Lee NH, Aberrant Alternative Splicing in African American Prostate Cancer: novel driver of tumor aggressiveness and drug resistance, accepted, Nature Communications.</p>
                        <div data-doi="10.1038/ncomms15921" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>37. Han B, Park HK, Wang H, Panneerselvam J, Shen Y, Zhang J, Li L, Lee YH, Su M, <strong>Ching T, Garmire LX </strong>, Jia W, Yu H, Fei P, HDBR1 Modulates U2 snRNP Function to Maintain RNA Populations, Contributing to the Suppression of Human Cancer Development, accepted, Oncogene.</p>
                        <div data-doi="10.1038/onc.2017.150" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>36. Greene CS, <strong> Garmire LX </strong>, Gilbert JA, Ritchie DR, Hunt L, Celebrate parasites, accepted, Nature Genetics. (
                            <a href="https://www.ncbi.nlm.nih.gov/pubmed/28358134">link</a>)
                        </p>
                        <div data-pmid="28358134" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>35. Feng N, Wang Y, Zheng M, Yu X, Lin H, Ma RN, Shi O, Zheng X, Gao M, Yu H, <strong>Garmire L</strong>, Qian B. Genome-wide analysis of DNA methylation and their associations with long noncoding RNA/mRNA expression in non-small-cell lung cancer. Accepted, Epigenomics. (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/28111977'>link</a>)
                        </p>
                        <div data-pmid="28111977" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <h4><strong>2016</strong></h4>

                    <div class="publication-item">
                        <p>34. Yang J, Tanaka Y, Seay M, Li Z, Jin JQ, <strong> Garmire L,
            Zhu X </strong>, Euskirchen G, Synder M, Li W, Park IH, Pan X, Weissman SM. Single cell transcriptomics reveals unanticipated features of early hematopoietic precursors. Nucleic Acids Research (2016). (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/28003475'>link</a>)
                        </p>
                        <div data-pmid="28003475" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>33. <strong>Zhu X, Ching T</strong>, Pan X, Weissman S,
                            <strong>Garmire LX</strong>. Detecting heterogeneity in single-cell RNA-Seq data by non-negative matrix factorization, PeerJ 5:e2888 (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/28133571'>link</a>)
                        </p>
                        <div data-pmid="28133571" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>32. <strong>Garmire LX</strong>, Gliske S, Nguyen QC, Chen JH, Nemati S, VAN Horn JD, Moore JH, Shreffler C, Dunn M. The training of next generation data scientists in biomedicine. Pac Symp Biocomput. 2016;22:640-645. (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/27897014'>link</a>)
                        </p>
                        <div data-pmid="27897014" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>31. Feng N§, <strong>Ching T</strong>§, Wang Y, Liu B, Lin H,Shi O,Zhang X, Yao Y, Hua L, Zheng X, Gao M, Yu H#, <strong>Garmire LX
            </strong>#, Qian B#. Analysis of Microarray Data on Gene Expression and Methylation to Identify Long Non-coding RNAs in Non-small Cell Lung Cancer. Scientific Reports 6 (2016). (#: co-corresponding authors) (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/27849024'>link</a>)
                        </p>
                        <div data-pmid="27849024" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>30. <strong>Lu L</strong>, McCurdy S, <strong>Huang S</strong>,
                            <strong>Zhu X</strong>, Peplowska K, Tiirikainen M, Boisvert WA,
                            <strong>Garmire LX</strong>, Time Series miRNA-mRNA integrated analysis reveals critical miRNAs and targets in macrophage polarization. Scientific Reports 6 (2016). (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/27981970'>link</a>)
                        </p>
                        <div data-pmid="27981970" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>29. <strong>Poirion A§, Zhu X§, Ching T, Garmire LX</strong> Single-Cell Transcriptomics Bioinformatics and Computational Challenges,7:163.Frontiers in Genetics (2016). (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/27708664'>link</a>)
                        </p>
                        <div data-pmid="27708664" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>28. <strong>Huang S*</strong>, Kou L*, Furuya H, Yu CH, Kattan M, Goodison S, <strong>Garmire LX</strong>, Rosser CJ, A nomogram derived by combination of demographic and biomarker data improves the non-invasive evaluation of patients at risk for bladder cancer, accepted, Cancer Epidemiology, Biomarkers and Prevention, 2016 Jul 6. pii: cebp.0260.
                        </p>
                        (<a href='http://www.ncbi.nlm.nih.gov/pubmed/27383773'>link</a>)
                        <div data-pmid="27383773" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>27. <strong>Wei R</strong>, De Vivo I, <strong>Huang S</strong>,
                            <strong>Zhu X</strong>, Risch, H, Moore JH, Yu H, <strong>Garmire
              LX</strong>, Meta-dimensional data integration identifies critical pathways for susceptibility, tumorigenesis and progression of endometrial cancer, <em>Oncotarget</em>, 2016 Jul 9. doi: 10.18632/oncotarget.10509. (
                            <a href='http://www.ncbi.nlm.nih.gov/pubmed/27409342'>link</a>)
                        </p>
                        <div data-pmid="27409342" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>26. <strong>Ching T</strong>, Peplowska K, <strong> Huang S, Zhu
              X </strong>, Shen Y, Molnar J, Yu H, Tiirikainen M, Fogelgren B, Fan R, <strong> Garmire LX </strong>. Pan-cancer analyses reveal a panel of biologically and clinically relevant lincRNAs for tumour diagnosis, subtyping and prognosis, 2016, 7:62-72, EBioMedicine. (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/27322459'>link</a>)
                        </p>
                        <div data-pmid="27322459" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>25. <strong>Huang S, Chong N</strong>, Lewis NE, Jia W, Xie G,
                            <strong>Garmire LX</strong>. Novel personalized pathway-based metabolomics models reveal key metabolic pathways for breast cancer diagnosis, 8(1):34, Genome Medicine (2016). (
                            <a href='http://www.ncbi.nlm.nih.gov/pubmed/27036109'>link</a>)
                        </p>
                        <div data-pmid="27036109" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <h4><strong>2015</strong></h4>

                    <div class="publication-item">
                        <p>24. <strong> Ching T, Masaki J, Weirather J, Garmire LX
            </strong>. Non-coding yet non-trivial: a review on the genomics of long intergenic non-coding RNAs, 8:44. doi: 10.1186/s13040-015-0075-z, BioData Mining. (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/26697116'>link</a>)
                        </p>
                        <div data-pmid="26697116" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>23. Xie G, Zhou B, Zhao Y, Qiu Y, Zhao X, <strong> Garmire LX
            </strong>, Yu H, Yen Y, Jia W, Lowered circulating aspartate is a metabolic feature of human breast cancer,Oncotarget, 6 (32), 33369-81. (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/26452258'>link</a>)
                        </p>
                        <div data-pmid="26452258" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>22. <strong>Li J</strong>, <strong>Ching T</strong>,
                            <strong>Huang S</strong>, <strong>Garmire LX</strong>, Using Epigenomics Data to Predict Differential Gene Expression in Lung Cancer, BMC Bioinformatics. 2015;16 Suppl 5:S10 (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/25861082'>link</a>)
                        </p>
                        <div data-pmid="25861082" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>21. <strong>Ching T, Ha J</strong>,Song MA, Tiirikainen M, Molnar J Berry M, Towner D, <strong>Garmire LX</strong>, Genome-scale hypomethylation in the cord blood cells associated with early onset preeclampsia, Clinical Epigenetics. 2015 Mar 13;7(1):21. (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/25806090'>link</a>)
                        </p>
                        <div data-pmid="25806090" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>20. Gagliani N, Iseppon A, Vesely CA, Brockmann L, Palm NW, Zeote MR, Licona-Limon P, Paiva R, <strong>Ching T</strong>, Zi X, Fan R, <strong>Garmire L</strong>, Geginat J, Stockinger B, Esplugues E, Huber S, Flavell R, Th17 cells transdifferentiate into regulatory T cells during resolution of inflammation, Nature. 2015 Apr 29. doi: 10.1038/nature14452. (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/25924064'>link</a>)
                        </p>
                        <div data-pmid="25924064" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <h4><strong>2014</strong></h4>

                    <div class="publication-item">
                        <p>19. <strong>Menor M</strong>, <strong>Ching T</strong>, Garmire D, <strong>Zhu X</strong>, <strong>Garmire
              LX</strong>, mirMark: a site-level and UTR-level classifier for miRNA target prediction. Genome Biology. Oct 25;15(10):500. (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/25344330'>link</a>)
                        </p>
                        <div data-pmid="25344330" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>18. Han L, Zi XY, <strong>Garmire LX</strong>, Pan XH, Weissman SM, Fan R, Co-detection and sequencing of genes and transcripts from the same single cells enabled by a microfluidics platform, Sci Rep. Sep 26;4:6485. doi: 10.1038/srep06485. (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/25255798'>link</a>)
                        </p>
                        <div data-pmid="25255798" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>17. <strong>Ching T</strong>, <strong>Huang S</strong>,
                            <strong>Garmire LX</strong>: power analysis and sample size estimation for RNA-Seq differential expression, RNA. 2014 Sep 22. [Epub ahead of print]. (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/25246651'>link</a>)
                        </p>
                        <div data-pmid="25246651" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>16. <strong>Huang S</strong>, Yee C, <strong>Ching T</strong>, Yu H, <strong>Garmire LX</strong>, A novel model to combine clinical and pathway-based transcriptomic information for the prognosis prediction of breast cancer, PLOS Computational Biology. Sep 18;10(9):e1003851. (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/25233347'>link</a>)
                        </p>
                        <div data-pmid="25233347" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>15. <strong>Ching T</strong>,Song MA, Tiirikainen M, Berry M, Towner D, <strong>Garmire LX</strong>, Global hypermehtylation coupled with promoter hypomethylation in the chorioamniotic membranes of early onset preeclampsia. Mol. Hum. Reprod. Sep;20(9):885-904. (
                            <a href='https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4371797/'>link</a>)
                        </p>
                        <div data-pmid="27322459" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <h4><strong>2013</strong></h4>

                    <div class="publication-item">
                        <p>14. <strong>Garmire LX</strong>, Subramaniam S. The poor performance of TMM on microRNA-Seq. RNA 19, 735-6 (2013). PMCID: PMC3683907. (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/23616641'>link</a>)
                        </p>
                        <div data-pmid="23616641" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>13. Hadd AG, Houghton J, Choudhary A, Sah S, Chen L, Marko AC, Sanford T, Buddavarapu K, Krosting J, <strong>Garmire LX</strong>, Wylie D, Shinde R, Beaudenon S, Alexander EK, Mambo E, Adai AT, Latham GJ. Targeted, high-depth, next-generation sequencing of cancer genes in formalin-fixed, paraffin-embedded and fine-needle aspiration tumor specimens. J Mol Diagn 15, 234-47 (2013). (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/23321017'>link</a>)
                        </p>
                        <div data-pmid="23321017" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <h4><strong>Pre-Garmire Group time. 2012 and earlier </strong></h4>

                    <div class="publication-item">
                        <p>12. Colas A, McKeithan W, Cunningham T, Bushway P, <strong>Garmire
              LX</strong>, Duester G, Subramaniam S, Mercola M, Whole genome microRNA screening identifies let-7 And mir-18 as regulators of germ layer formation during early embryogenesis, Genes &amp; Development 26, 2567-79 (2012).PMCID: PMC3521625. (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/23152446'>link</a>)
                        </p>
                        <div data-pmid="23152446" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>11. Wu Y, <strong>Garmire LX</strong>, Fan R, Dynamic analysis of intercellular signaling reveals a mechanistic transition in tumor microenvironment, Integrative Biology,(Camb) 4, 1478-86 (2012).PMCID: PMC3502715. (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/23080410'>link</a>)
                        </p>
                        <div data-pmid="23080410" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>10. Nathan S*, <strong>Garmire LX</strong>*, McDonald J, Norihito S, Reichart D, Heudobler D, Raetz CR, Murphy RC, Merril AH, Brown A, Dennis EA, Li AC, Fahy E, Subramaniam S, Quehenberger O, Russell DW, and Glass CK, Regulated accumulation of desmosterol integrates macrophage lipid metabolism and inflammatory responses. Cell 151, 138-52 (2012) (*: equal contribution). PMCID: PMC3464914. (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/23021221'>link</a>)
                        </p>
                        <div data-pmid="23021221" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>9. <strong>Garmire LX</strong>, Subramaniam S. Evaluation of microRNA-Seq normalization methods. RNA 18, 1279-1288 (2012). PMCID: PMC3358649. (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/22532701'>link</a>)
                        </p>
                        <div data-pmid="22532701" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>8. <strong>Garmire LX</strong>, Garmire DG, Huang W, Yao J, Glass CK, Subramaniam S. A global clustering approach to identify intergenic non-coding RNA, with application in mouse macrophages. PLoS ONE 6(9):e24051 (2011). PMCID: PMC3184070. (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/21980340'>link</a>)
                        </p>
                        <div data-pmid="21980340" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>7. Wang KC, <strong>Garmire LX</strong>, Young A, Nguyen P, Trinh A, Subramaniam S, Wang NP, Shyy J, Li J, Chien S, Role of miR-23b in flow-regulation of microRNA signature and cell growth in endothelial Cells, Proc Natl Acad Sci U S A 107, 3234-9 (2010). PMCID: PMC2840325. (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/20133741'>link</a>)
                        </p>
                        <div data-pmid="20133741" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>6. <strong>Garmire LX</strong>, Shen ZX, Briggs S, Yeo G, Glass CK, Subramaniam S, Regulatory Network of microRNAs in RAW 264.7 Macrophage Cells. Proceedings of 32nd International Conference of the IEEE Eng Med Biol Soc 6198-201 (2010). (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/21097158'>link</a>)
                        </p>
                        <div data-pmid="21097158" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>5. <strong>Garmire LX</strong>, Hunt CA, In silico methods for unraveling the mechanistic complexities of intestinal absorption: metabolism-efflux transport interactions. Drug Metab Dispos 36,1414-24 (2008). PMC, in process. (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/18436618'>link</a>)
                        </p>
                        <div data-pmid="18436618" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>4. <strong>Garmire LX</strong>, Garmire DG, Hunt CA, An in silico transwell device for drug transport and drug-drug interaction studies. Pharmaceutical Research 24, 2171-86 (2007). Featured Article. (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/17703347'>link</a>)
                        </p>
                        <div data-pmid="17703347" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>3. <strong>Garmire LX</strong>, Mechanistic study of enzyme-efflux transporter relations using in silico devices. Lecture Notes in Engineering and Computer Science 2167, 34-39 (2007). (
                            <a href='http://www.iaeng.org/publication/WCECS2007/WCECS2007_pp34-39.pdf'>link</a>)
                        </p>
                    </div>

                    <div class="publication-item">
                        <p>2. Grant MR, Hunt CA, <strong>Xia L</strong>*, Fata JM, Bissell MJ, Modeling mammary gland morphogenesis as a reaction-diffusion process, Proceedings of the 26th Annual International Conference of the IEEE EMBS, San Francisco, CA, USA September 1-5 (2004) (*: Maiden name). (
                            <a href='https://www.ncbi.nlm.nih.gov/pubmed/17271768'>link</a>)
                        </p>
                        <div data-pmid="17271768" data-badge-popover="bottom" data-badge-type="donut" data-hide-no-mentions="true" class="altmetric-embed"></div>
                    </div>

                    <div class="publication-item">
                        <p>1. Fan T, <strong>Xia L</strong>*, Han Y. Mitochondrion and apoptosis. Acta Biochica et Biophysica Sinica 33, 7-12 (2001) (*: Maiden name). (
                            <a href='https://www.researchgate.net/publication/11246090_Mitochondria_and_Apoptosis'>link</a>)
                        </p>
                    </div>
                </div>
            </div>
            <div class="tab-pane fade" id="software">
                <div id="mainContent">
                    <h2>Software</h2>
                    <ul>
                        <li><a href="https://github.com/lanagarmire/cox-nnet">cox-nnet</a> - predict patient prognosis by extending Cox Regression to the non-linear neural network framework</li>
                        <li><a href="https://github.com/lanagarmire/SSrGE">SSrGE</a> - Using Single Nucleotide Variations in Single-Cell RNA-Seq to Identify Tumor Subpopulations and Genotype-phenotype Linkage</li>
                        <li><a href="https://github.com/lanagarmire/lilikoi">Lilikoi</a> - an R package for personalized pathway-based classification modeling using metabolomics data</li>
                        <li><a href="http://garmiregroup.org/granatum/app">Granatum</a> - a single-cell RNA-Seq analysis platform for bench scientists </li>
                        <li><a href="http://garmiregroup.org/granatum/NMFEM">NMFEM</a> - cluster scRNA-Seq samples using NMF, extract important genes, and generate PPI network modules from these genes </li>
                        <li><a href="http://mn.eng.hawaii.edu/~garmire/HTML/MirMark/">mirMark</a> - random forest based machine learning method to predict the microRNA targets both at site-level and UTR-level </li>
                        <li><a href="http://www2.hawaii.edu/~lgarmire/RNASeqPowerCalculator.htm">RNASeqPowerCalculator</a> - R code to calculate the power and sample size for RNA-Seq differential expression </li>
                        <li><a href="http://www2.hawaii.edu/~lgarmire/MetaboloPathwayModel.html">MetaboloPathwayModel</a> - R code to build pathway-based metabolomics diagnosis models </li>
                    </ul>
                </div>
            </div>
        </div>
        <div class="row">
            <div class="col-md-12">
                <div class="footer">
                    <span class="pull-left">
              <span class="info-phrase">Garmire Group @ 2014</span> |
                    <span class="info-phrase">LGarmire@cc.hawaii.edu</span> |
                    <span class="info-phrase">(808) 441-8193</span> |
                    <span class="info-phrase">University of Hawaii Cancer Center, 701 Ilalo St, Honolulu, HI 96813</span> |
                    <span class="info-phrase" id="visitor-counter"></span>
                    </span> <span class="pull-right">Pages designed by Xun Zhu</span></div>
            </div>
        </div>
    </div>
    <script src="http://code.jquery.com/jquery-1.11.0.min.js"></script>
    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
    <script src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
    <script>
        var fhsh = document.createElement('script');
        var fhs_id_h = "3131394";
        fhsh.src = "//s1.freehostedscripts.net/ocount.php?site=" + fhs_id_h + "&name=Visits&a=1";
        document.head.appendChild(fhsh);
        document.getElementById('visitor-counter').innerHTML = "<span id='h_" + fhs_id_h + "'></span>";

        (function(i, s, o, g, r, a, m) {
            i['GoogleAnalyticsObject'] = r;
            i[r] = i[r] || function() {
                (i[r].q = i[r].q || []).push(arguments)
            }, i[r].l = 1 * new Date();
            a = s.createElement(o),
                m = s.getElementsByTagName(o)[0];
            a.async = 1;
            a.src = g;
            m.parentNode.insertBefore(a, m)
        })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
        ga('create', 'UA-54742909-1', 'auto');
        ga('require', 'displayfeatures');
        ga('send', 'pageview');
    </script>
</body>

</html>
